scrooge-core/src/main/scala/com/twitter/scrooge/ThriftEnum.scala

```diff
@@ -5,5 +5,6 @@ import org.apache.thrift.TEnum
 trait ThriftEnum extends TEnum {
   def value: Int
   def name: String
+  def originalName: String
```

Comment:
can we scaladoc what `originalName` represents? 

i'm going to forget by the next time i look at this code.